oak-doc/src/site/markdown/development/nodestore-fixtures.md

```diff
@@ -0,0 +1,44 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+  -->
+
+Testing using different NodeStore fixtures
+---------------
+
+In Jackrabbit Oak unit and integration tests are run using different fixtures which represent configurations of the NodeStore underlying storage for the content repository.
+This ensures that there is a fixed environment in which tests are run and the results are consistent and repeatable across different runs.
+
+The default fixture used when running `mvn clean install` is `SEGMENT_TAR` which uses the SegmentNodeStore with the TarMK backend. 
+
+To run the tests using a different fixture, you can use the following command: `mvn clean install -Dfixture=<FIXTURE>`. 
+You can also update the `pom.xml` file to use a different fixture by updating the `fixture` property in the `oak-parent/pom.xml` file. If you want to run using multiple fixtures, provide a space separated list of fixtures.
+
+These are the possible fixtures that can be used:
+- `SEGMENT_TAR` - Uses the SegmentNodeStore with the TarMK backend.
```

Comment:
Not sure we want to use TarMK as a term here as it is somewhat outdated - what about "local storage with TAR files" (as is used [here](https://jackrabbit.apache.org/oak/docs/nodestore/segment/overview.html#remote-segment-stores)), or something similar?

Reply:
done :)